content/articles/how-to-work-with-lombok-on-quarkus/index.md

```diff
@@ -0,0 +1,837 @@
+│   └── README.md
+```
+
+- Delete the files found inside the Java, **resources, and test folders.
+
+### Create a Movies sample project
+
+Create a new `Movie.java` file inside the java folder and in the file, add the following snippets.
+
+```Java
+import java.net.URL;
+
+/**
+* The type Movie.
+  */
+  public class Movie {
+  private Long id;
+  private String title;
+  private String description;
+  private String country;
+  private int rating;
+  private URL officialSite;
+  private String language;
+
+  /**
+  * Instantiates a new Movie.
+    */
+    public Movie(Long id, String title, String description, String country, int rating, URL officialSite, String language) {
+    this.id = id;
+    this.title = title;
+    this.description = description;
+    this.country = country;
+    this.rating = rating;
+    this.officialSite = officialSite;
+    this.language = language;
+    }
+
+  /An empty constructor/
+  public Movie() {
+
+  }
+
+  /Set Getters and Setters/
+  public Long getId() {
+  return id;
+  }
+
+  public void setId(Long id) {
+  this.id = id;
+  }
+
+  public String getTitle() {
+  return title;
+  }
+
+  public void setTitle(String title) {
+  this.title = title;
+  }
+
+  public String getDescription() {
+  return description;
+  }
+
+  public void setDescription(String description) {
+  this.description = description;
+  }
+
+  public String getCountry() {
+  return country;
+  }
+
+  public void setCountry(String country) {
+  this.country = country;
+  }
+
+  public int getRating() {
+  return rating;
+  }
+
+  public void setRating(int rating) {
+  this.rating = rating;
+  }
+
+  public URL getOfficialSite() {
+  return officialSite;
+  }
+
+  public void setOfficialSite(URL officialSite) {
+  this.officialSite = officialSite;
+  }
+
+  public String getLanguage() {
+  return language;
+  }
+
+  public void setLanguage(String language) {
+  this.language = language;
+  }
+  }
+```
+
+In the '`pom.xml`' file, add the code below that adds the Jupiter JUnit package under the dependencies section. This dependency allows testing of the project and running of additional test units.
+
+```xml
+<!--junit-jupiter.version-->
+<dependency>
+    <groupId>org.junit.jupiter</groupId>
+    <artifactId>junit-jupiter-engine</artifactId>
+    <version>${junit-jupiter.version}</version>
+    <scope>test</scope>
+</dependency>
+```
+
+Add another code before the dependencies section as follows:
+
+```xml
+<properties>
+<maven.compiler.source>17</maven.compiler.source>
+<maven.compiler.target>17</maven.compiler.target>
+<junit-jupiter.version>5.8.1</junit-jupiter.version>
+<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+</properties>
+```
+
+This section holds some variables which state the versions of the dependencies used. For example, that of `JUnit` is `5.8.1`. First, reload the IDE or project to apply the changes made, then in the next step, create a new file inside the `test` folder. Name it `MovieTest.java` file.
```

Comment:
Move this para before the code is introduced.

Reply:
Good point!